### DISCUSSION TITLE
17.0.0

### DISCUSSION THREAD

**Discussion by jamesdaniels** [angular/angularfire#3475] [General]
* Mark as compatible with ng@17
* Bump firebase dependency
* Add configurations to ng-deploy
* Utilize firebase-tools 13+
* Work arounds for bugs in firebase-tools

<hr /><em>This discussion was created from the release <a href='https://github.com/angular/angularfire/releases/tag/17.0.0'>17.0.0</a>.</em>

**Comment by sorcamarian** [comment]
I use version ^7.5.0 of angularfire and version 9 of firebase.
Do I need to modify code when moving to the current version 17?

**Reply by jamesdaniels** [reply]
There should not be any breaking changes, you should be able to just bump the dependency
